### PR TITLE
Improve Concatenation Strings Descriptions

### PR DESCRIPTION
**Pull request by SaintPeter**
Added a note about watching out for spaces, to reduce confused questions that miss that trailing space.

image

Tested Locally.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -951,6 +951,7 @@
         "In JavaScript, when the <code>+</code> operator is used with a <code>String</code> value, it is called the <dfn>concatenation</dfn> operator. You can build a new string out of other strings by <dfn>concatenating</dfn> them together.",
         "<strong>Example</strong>",
         "<blockquote>'My name is Alan,' + ' I concatenate.'</blockquote>",
+        "<strong>Note</strong><br>Watch out for spaces.  Concatenation does not add spaces between concatenated strings, so you'll need to add it yourself.",
```
Comment (Rafase282) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: I think it should stay as plural. "... so you will need to add *them* yourself.

Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: I agree.  I'll fix that and re-push.

**Comment by SaintPeter** [comment]
Changed "it" to "them".

**Comment by ltegman** [comment]
:+1: Good catch on the pattern of questions on this!

**Pull request merged**